packages/kbn-scout-info/src/reporting.ts

```diff
@@ -7,6 +7,21 @@
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
 
-export const SCOUT_TEST_EVENTS_TEMPLATE_NAME = 'scout-test-events';
-export const SCOUT_TEST_EVENTS_INDEX_PATTERN = `${SCOUT_TEST_EVENTS_TEMPLATE_NAME}-*`;
-export const SCOUT_TEST_EVENTS_DATA_STREAM_NAME = `${SCOUT_TEST_EVENTS_TEMPLATE_NAME}-kibana`;
+function booleanFromEnv(varName: string, defaultValue: boolean = false): boolean {
```

Comment:
I believe this is not behaving exactly as you'd expect. When a `defaultValue = true` is set, you can not have this function evaluate to false. You probably need to treat the missing / incorrect values as a third category (besides true/false) and use the default value only then.

Reply:
Resolved in https://github.com/elastic/kibana/pull/206431/commits/77508b2fed51290c371f12dc29c8e908ed8a3e59